src/app/zap-templates/zcl/data-model/chip/target-navigator-cluster.xml

```diff
@@ -51,4 +51,9 @@ limitations under the License.
       <item name="identifier" type="INT8U"/>
       <item name="name" type="OCTET_STRING" length="32"/> <!-- Change this to CHAR_STRING once it is supported #6112 -->
   </struct>
+
+  <struct name="TargetNavigatorResponse">
```

Comment:
Again, I don't see such a struct in the spec.

Reply:
Same as previous two. And same as previous two i've removed this and added a new header file which now holds the needed data structure.